packages/validate/src/ValidateMixin.js

```diff
@@ -24,7 +26,15 @@ function arrayDiff(array1 = [], array2 = []) {
 export const ValidateMixin = dedupeMixin(
   superclass =>
     // eslint-disable-next-line no-unused-vars, no-shadow
-    class ValidateMixin extends SyncUpdatableMixin(SlotMixin(superclass)) {
+    class ValidateMixin extends ScopedElementsMixin(SyncUpdatableMixin(SlotMixin(superclass))) {
+      static get scopedElements() {
+        return {
+          ...super.scopedElements,
+          // TODO: Remove once we can load it asynchronously, see todo below in loadFeedbackComponent method
```

Comment:
we no longer need to load it asynchronously as we can override it when extending

Reply:
True! Good catch :D